### PR TITLE
[Tornado] Get rid of Toro

### PR DESCRIPTION
**Pull request by noxiouz**

### CODE REVIEW

File: cocaine/proxy/proxy.py [hunk 1]
```diff
@@ -440,9 +439,6 @@ def process(self, request, name, app, event, data):
 
         parentid = 0
 
-        # We generate the start trace here, as nginx right now cannot send proper initial traceid.
-        # So both traceid & spaid are both random and the same,
-        # parentid = 0 (means the root of trace)
```
Comment (3Hren) [inline cocaine/proxy/proxy.py]: Why did you remove comments?

Comment (noxiouz) [inline cocaine/proxy/proxy.py]: It's valid anymore, as nginx generates good traceid
